src/eureka/S3_data_reduction/plots_s3.py

```diff
@@ -426,3 +427,79 @@ def driftywidth(data, meta):
     plt.savefig(meta.outputdir+fname, bbox_inches='tight', dpi=300)
     if not meta.hide_plots:
         plt.pause(0.2)
+
+
+def residualBackground(data, meta, m):
+    '''Plot the median, BG-subtracted frame to study the residual BG region and
+    aperture/BG sizes. (Fig 3304)
+
+    Parameters
+    ----------
+    data : Xarray Dataset
+        The Dataset object.
+    meta : eureka.lib.readECF.MetaClass
+        The metadata object.
+    m : int
+        The file number.
+
+    Returns
+    -------
+    None
+
+    Notes
+    -----
+    History:
+
+    - 2022-07-29 KBS
+        First version
+    '''
+    # Median flux of segment
+    flux = np.median(data.flux, axis=0)
+    # Compute vertical slice of with 10 columns
+    slice = np.nanmedian(flux[:, meta.subnx//2-5:meta.subnx//2+5], axis=1)
+    # Interpolate to 0.01-pixel resolution
+    f = spi.interp1d(np.arange(meta.subny), slice, 'cubic')
+    ny_hr = np.arange(0, meta.subny-1, 0.01)
+    flux_hr = f(ny_hr)
+
+    vmin = -200
+    vmax = 1000
```

Comment:
Why these values in particular?

Reply:
They work well for visualizing the BG.  Made them optional parameters, in case we need to change them per instrument.